datajunction-clients/python/datajunction/models.py

```diff
@@ -121,6 +122,7 @@ class Tag(BaseModel):
     name: str
     display_name: str
     tag_type: str
+    created_by: int = Field(foreign_key="users.id")
```

Comment:
Should this be a `created_by_id: int`, and then there be an additional Relationship model like:
```
created_by: User = Relationship(
    sa_relationship_kwargs={
        "lazy": "joined",
    },
)
```

This would make it so that the user object can be accessed via `created_by`

Reply:
Oh yeah, that's definitely better, I'll update this